frameworks/C++/cuehttp/benchmark_config.json

```diff
@@ -0,0 +1,25 @@
+{
+  "framework": "cuehttp",
+  "tests": [
+    {
+      "default": {
+        "plaintext_url": "/plaintext",
+        "port": 10001,
```

Comment:
Can you update this port to something normally used for http-alt, like 8080? (We match over some of these common ports for debugging)